firmware/fsm.c

```diff
@@ -292,12 +292,14 @@ void fsm_msgFirmwareUpload(FirmwareUpload *msg)
 
 void fsm_msgGetEntropy(GetEntropy *msg)
 {
+#if DEBUG_LINK == 0
```

Comment:
What do you think about not using DEBUG_LINK here, but rather introducing another option, such as DEBUG_RNG?